var/spack/repos/builtin/packages/intel-oneapi-ippcp/package.py

```diff
@@ -11,7 +11,18 @@
 
 @IntelOneApiPackage.update_description
 class IntelOneapiIppcp(IntelOneApiLibraryPackage):
-    """Intel oneAPI IPP Crypto."""
+    """Intel Integrated Performance Primitives (Intel IPP) is an extensive
+       library of ready-to-use, domain-specific functions that are
+       highly optimized for diverse Intel architectures. These
+       functions take advantage of Single Instruction, Multiple Data
+       (SIMD) instructions and improve the performance of
+       computation-intensive applications, including signal
+       processing, data compression, video processing, and
+       cryptography. The intel-oneapi-ippcp package contains support
+       for cryptography and everything else can be found in the
+       intel-oneapi-ipp package.
+
+    """
```

Comment:
It is the same product, but 2 packages. Marketing wanted one description. I added that line near the end explaining the 2 packages. I can change that sentence or move it to the beginning if you think it would be better.

Reply:
Not at all.  I just wanted to confirm it was not an accidental copy-and-paste kind of issue.

Reply:
Then I will leave it as-is.